packages/actor-rdf-resolve-hypermedia-links-queue-wrapper-info-occupancy/README.md

```diff
@@ -0,0 +1,40 @@
+# Comunica Wrapper Info Occupancy RDF Resolve Hypermedia Links Queue Actor
+
+[![npm version](https://badge.fury.io/js/%40comunica%2Factor-rdf-resolve-hypermedia-links-queue-wrapper-info-occupancy.svg)](https://www.npmjs.com/package/@comunica/actor-rdf-resolve-hypermedia-links-queue-wrapper-info-occupancy)
+
+A comunica Wrapper Info Occupancy RDF Resolve Hypermedia Links Queue Actor.
+
+This module is part of the [Comunica framework](https://github.com/comunica/comunica),
+and should only be used by [developers that want to build their own query engine](https://comunica.dev/docs/modify/).
+
+[Click here if you just want to query with Comunica](https://comunica.dev/docs/query/).
+
+This link queue wrapper's purpose is to collect information about the occupancy of the link queue in a file.
+It should not be used in production or in a browser environment.
+To observe non-standard properties of the link queue the `IOptionalLinkQueueParameters` can be extended and the methods can be modified to process those parameters.
```

Comment:
Can you link to https://github.com/constraintAutomaton/comunica-link-queue-parser-rs so readers of this readme can know how they should get and process the output when enabling this actor?